README.md

```diff
@@ -62,6 +64,7 @@ const iTwinPlugin = require("@itwin/eslint-plugin");
 module.exports = [
   {
     files: ["**/*.{ts,tsx}"],
+    languageOptions: iTwinPlugin.languageOptions.typescript,
```

Comment:
Can't you just do this...
```js
{
    files: ["**/*.{ts,tsx}"],
    ...iTwinPlugin.configs.iTwinjsRecommendedConfig,
    languageOptions: customLanguageOptions,
  },
```
...when you need to override the defaults? You wouldn't have to change the plugin at all and 99% of our users wouldn't need to add an extra line to their config.